plugins/Jobs.cpp

```diff
@@ -44,6 +44,10 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
     STable& content = command.jsonContent;
     const string& requestVerb = request.getVerb();
 
+    // Each command is unique, so if the command causes a crash, we'll identify it on a unique random number.
+    command.request["crashID"] = to_string(SRandom::rand64());
```

Comment:
I don’t think this is going to work... this is it going to prevent the same bad command from crashing multiple nodes.

Reply:
Okay so i rethought about this, and i was wrong.